src/ui/public/visualize/loader/visualize_data_loader.ts

```diff
@@ -117,28 +114,7 @@ export class VisualizeDataLoader {
         },
       };
     } catch (error) {
-      params.searchSource.cancelQueued();
-
-      this.vis.requestError = error;
-      this.vis.showRequestError =
-        error.type && ['NO_OP_SEARCH_STRATEGY', 'UNSUPPORTED_QUERY'].includes(error.type);
-
-      // tslint:disable-next-line
-      console.error(error);
-
-      if (isTermSizeZeroError(error)) {
```

Comment:
I removed `isTermSizeZeroError` and the corresponding `elasticsearch_errors` directory because it is no longer possible to set a term size: 0 in the UI. And if you add it elsewhere, e.g. URL params, you will get a different error before this line of code even executes.

(This is the only place `isTermSizeZero` error is used)